pkg/router/controller/host_admitter_test.go

```diff
@@ -500,14 +506,17 @@ func TestValidRouteAdmissionFuzzing(t *testing.T) {
 	}
 
 	routes := []*routeapi.Route{
-		makeRoute("ns1", "r1", "net", false, makeTime(0*time.Second)),
-		makeRoute("ns2", "r2", "com", false, makeTime(1*time.Second)),
-		makeRoute("ns3", "r3", "domain1.com", false, makeTime(2*time.Second)),
-		makeRoute("ns4", "r4", "domain2.com", false, makeTime(3*time.Second)),
-		makeRoute("ns5", "r5", "foo.domain1.com", false, makeTime(4*time.Second)),
-		makeRoute("ns6", "r6", "bar.domain1.com", false, makeTime(5*time.Second)),
-		makeRoute("ns7", "r7", "sub.foo.domain1.com", true, makeTime(6*time.Second)),
-		makeRoute("ns8", "r8", "sub.bar.domain1.com", true, makeTime(7*time.Second)),
+		makeRoute("ns1", "r1", "net", "", false, makeTime(0*time.Second)),
+		makeRoute("ns2", "r2", "com", "", false, makeTime(1*time.Second)),
+		makeRoute("ns3", "r3", "domain1.com", "", false, makeTime(2*time.Second)),
+		makeRoute("ns4", "r4", "domain2.com", "", false, makeTime(3*time.Second)),
+		makeRoute("ns5", "r5", "foo.domain1.com", "", false, makeTime(4*time.Second)),
+		makeRoute("ns6", "r6", "bar.domain1.com", "", false, makeTime(5*time.Second)),
+		makeRoute("ns7", "r7", "sub.foo.domain1.com", "", true, makeTime(6*time.Second)),
+		makeRoute("ns8", "r8", "sub.bar.domain1.com", "", true, makeTime(7*time.Second)),
+		makeRoute("ns8", "r8", "sub.bar.domain1.com", "/p1", true, makeTime(8*time.Second)),
+		makeRoute("ns8", "r8", "sub.bar.domain1.com", "/p2", true, makeTime(9*time.Second)),
+		makeRoute("ns8", "r8", "sub.bar.domain1.com", "/p1/p2/p3", true, makeTime(10*time.Second)),
```

Comment:
give each route it's own name, and add tests for non-wildcard routes with the same host and different paths:

```
		makeRoute("ns8", "r8", "sub.bar.domain1.com", "", true, makeTime(7*time.Second)),
		makeRoute("ns8", "r9", "sub.bar.domain1.com", "/p1", true, makeTime(8*time.Second)),
		makeRoute("ns8", "r10", "sub.bar.domain1.com", "/p2", true, makeTime(9*time.Second)),
		makeRoute("ns8", "r11", "sub.bar.domain1.com", "/p1/p2/p3", true, makeTime(10*time.Second)),
		makeRoute("ns9", "r12", "sub.bar.domain2.com", "", false, makeTime(11*time.Second)),
		makeRoute("ns9", "r13", "sub.bar.domain2.com", "/p1", false, makeTime(12*time.Second)),
```